include/http/router.js

```diff
@@ -145,23 +151,41 @@ module.exports = function (pb) {
                     .use(this.router.allowedMethods());
 
                 this._addDefaultMiddleware();
-
-                this.__server = this.app.listen(port, () => {
-                    pb.log.info('PencilBlue is ready!');
-                });
+                if (this.useSSL()) {
+                    const config = {
```

Comment:
I would almost prefer to move this whole bit to a function.  

_startSSLServer() or something.  Not going to require that or anything.  This is much better than A.) It was earlier and B.) they way I had it... as the way I had it broke SSL support.  Opps.

Reply:
Aaand Done :P